src/package-manager.coffee

```diff
@@ -405,6 +406,14 @@ class PackageManager
     else
       Q.reject(new Error("Failed to load package '#{name}'"))
 
+  triggerActivationHook: (hook) ->
+    return unless hook? and _.isString(hook) and hook.length > 0
```

Comment:
Appreciate the caution here, but I wonder if we should blow up clearly rather than failing silently...